frontend/src/lib/MetricsManager.ts

```diff
@@ -54,6 +54,14 @@ export class MetricsManager {
    */
   private pendingDeltaCounter: DeltaCounter = {}
 
+  /**
+   * Report hash uniquely identifies "projects" so we can tell
```

Comment:
@jrhone, this is essentially what the issue is about.

Reply:
We want to add a report hash (which is a hash of installation id and script path) to our metrics.